index.html

```diff
@@ -401,9 +405,206 @@ <h2 class="component_example">Example For Simple Ratings</h2>
                 </div>
             </div>
             <hr>
+            <!-- input -->
+            <div id="input">
+                <h1 class="component_heading">Input</h1>
+                <p class="para">
+                    Input fields are an essential user interface design element.These input fileds used in many
+                    situation but most of people used for entring personal details,delivery address on e-commerce web
+                    forms.
+                </p>
+                <h2 class="component_example">Example simple input filed</h2>
+                <div>
+                    <p class="para">
+                        Inputs are used in forms to collect user related data.
+                    </p>
+                </div>
+                <div class="component_box input_box">
+                    <div>
+                        <label for="username" class="lable_text">Username *</label>
+                        <input type="text" name="text" class="user_text" placeholder="Enter your name">
```

Comment:
add Id="username" in Input , this specifies that both label and input are connected.

Reply:
ohh ok i wil change it